src/Handler/CurlFactory.php

```diff
@@ -370,8 +370,9 @@ private function applyHandlerOptions(EasyHandle $easy, array &$conf)
             $conf[CURLOPT_FILE] = fopen('php://temp', 'w+');
             $easy->sink = Psr7\stream_for($conf[CURLOPT_FILE]);
         }
-
+        $lessThan1s = false;
```

Comment:
can we call this, `$timeoutRequiresNoSignal` or something more specific? Can you also add a comment that describes why we are doing this and link to the original comment at http://php.net/manual/en/function.curl-setopt.php#104597